app/Providers/AppServiceProvider.php

```diff
@@ -153,5 +154,13 @@ public function boot(): void
         });
 
         Feature::discover();
+
+        if (! app()->runningUnitTests()) {
+            Table::configureUsing(function (Table $table): void {
+                $table
+                    ->paginationPageOptions([5, 10, 20, 'all'])
+                    ->defaultPaginationPageOption(5);
+            });
+        }
```

Comment:
I don't think we should only run this during non-testing.

The test state should be as close as possible to the Production state to which this deviates.

Let's remove the `app()->runningUnitTests()` here and modify the tests that need modifying to select a different pagination option if needed.